pp3/module/Application/src/Application/Entity/Plugin.php

```diff
@@ -116,6 +116,19 @@ private function updateVersions($versioning) {
         }
     }
 
+    public function removeAuthor($user) {
+        $this->authors->removeElement($user);
+    }
+
+    public function addAuthor($user) {
```

Comment:
Could return true/false if user was actually added or not which could then be reflected in both **Plugin/AdminControllers** by displaying "_Cannot add the same user twice._" information message.